Src/BootCamp.Chapter/BalanceStats.cs

```diff
@@ -112,30 +113,30 @@ public static string FindRichestPerson(string[] peopleAndBalances)
         {
             if (peopleAndBalances == null || peopleAndBalances.Length == 0) return "N/A.";
 
-            int currentTopBalance = 0;
-            List<String> listOfPeople = new List<String>();
+            decimal currentTopBalance = 0;
+            List<String> people = new List<String>();
 
             foreach (var values in peopleAndBalances)
             {
-                var array = values.Split(", ");
+                var array = values.Split(',').Select(a => a.Trim()).ToArray();
 
                 // Parse the last value of each person
-                bool succes = int.TryParse(array[^1], out int parsed);
+                bool parsedAmount = decimal.TryParse(array[^1], out decimal parsed);
```

Comment:
This name is better then `succes` but has a new problem! `parsedAmount`, unless you are looking at the exact line it's declared and set on looks like it would be holding a numeric value, an amount of some kind. In fact it holds the boolean result of the parsing attempt. Add an 'is' ir 'was' in there to make it clear that this variable holds the results of some other action. Something like `amountWasParsed` or similar along those lines would go a long way towards improving code clarity.